pkg/manager/member/tidb_member_manager.go

```diff
@@ -1044,6 +1054,93 @@ func (m *tidbMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 	return nil
 }
 
+const tidbSupportLabelsMinVersin = "6.2.0"
+
+func (m *tidbMemberManager) setServerLabels(tc *v1alpha1.TidbCluster) (int, error) {
+	if tc.Spec.TiDB.Version != nil && (*tc.Spec.TiDB.Version) != "" {
```

Comment:
If the client use the `image` field not `version`, the version is empty. How about adding a function `TiDBVersion()` to parse version, like `TiFlashVersion`.
```go
// TiFlashVersion returns the image version used by TiFlash.
//
// If TiFlash isn't specified, return empty string.
func (tc *TidbCluster) TiFlashVersion() string {
	if tc.Spec.TiFlash == nil {
		return ""
	}

	image := tc.TiFlashImage()
	colonIdx := strings.LastIndexByte(image, ':')
	if colonIdx >= 0 {
		return image[colonIdx+1:]
	}

	return "latest"
}
```

In the future, we just need modify this function when we remove the `image` filed.

Reply:
done.